frb/igm.py

```diff
@@ -0,0 +1,465 @@
+""" Module for IGM calculations
```

Comment:
this module is now in `frb/dm`

Please remove it from here.  Thanks!

Reply:
Removed!